### PR TITLE
Auto publish to Maven

### PR DESCRIPTION
**Pull request by bootstraponline**

### CODE REVIEW

**Comment by winterDroid** [comment]
Amazing 👍

File: test_runner/build.gradle.kts [hunk 1]
```diff
@@ -11,6 +14,53 @@ plugins {
...
+        })
+    })
+}
+
+publishing {
+    publications {
+        register("mavenJava", MavenPublication::class) {
+            from(components["java"])
+
+            groupId = "flank"
```
Comment (runningcode) [inline test_runner/build.gradle.kts]: should the groupId include `testarmada` or some other unique identifier?

Comment (bootstraponline) [reply test_runner/build.gradle.kts]: I think flank is unique? I hesitate to use `testarmada` since I'm not sure that'll be true forever.

Comment (runningcode) [reply test_runner/build.gradle.kts]: Makes sense. Flank is an English word, I think its not the best to use english words as group ids. Usually it is a website identifier.

Anyways, the maven jar is working now! I really appreciate it!

**Pull request merged**